File: unusual-spending-ruby/lib/payments/payments.rb
class Payment
  def initialize
    @category =  ['fun', 'food', 'groceries', 'other'].sample
    @amount = rand(5.2...76.9)
  end
end

def get_payments(user_id, month)
  Array.new(rand(1..100)) { Payment.new }
end

File: unusual-spending-c-sharp/Payments.cs
using System;
using System.Collections.Generic;

namespace unusual_spending_c_sharp
{
    class Payments
    {
        public static List<Payment> fetch(string userId, int year, int month)
        {
            throw new NotImplementedException();
        }
    }
    
    struct Payment
    {
        double amount;
        string category;
        string description;
    }
}